docker/full-tls-mutinynet/docker-compose.yaml

```diff
@@ -241,6 +241,172 @@ services:
     restart: always
     platform: linux/amd64
 
+  prosody_config_writer:
```

Comment:
cant all instances of `fedimint.my-super-host.com` be written in as an environment variable or no?

Reply:
Not sure :thinking: